### DISCUSSION TITLE
SmartAutoClicker-2.4.1

### DISCUSSION THREAD

**Discussion by Nain57** [Nain57/Smart-AutoClicker#355] [Announcements]
Bug Fixes:

* Improve invalid native load and error display
* Fix another crash on Xiaomi. As this phone maker can't do a proper implementation of Android in its shitty devices, this is the last workaround i will make for those phones. 

<hr /><em>This discussion was created from the release <a href='https://github.com/Nain57/Smart-AutoClicker/releases/tag/2.4.1'>SmartAutoClicker-2.4.1</a>.</em>

**Comment by c-sanchez** [comment]
Heh I didn't expect it to give you problems Xiaomi.
I use Smart AutoClicker often and have not noticed any serious usage problems.

Although something that often happens is that for some strange reason the accessibility permissions are lost and I have to give them again.

![](https://i.ibb.co/bb8jpjk/SAC-Permision.jpg)
Is this what you fixed or was it something else?

**Reply by 12ChuAnh05** [reply]
You have to prevent system from stopping this app by disabling Battery Optimization and turning on Auto Run

**Reply by c-sanchez** [reply]
Thank you very much for your reply @19ChuAnh13 
I am going to try changing those options as you indicate :)
